src/mixxx.cpp

```diff
@@ -924,6 +932,23 @@ void MixxxMainWindow::slotHelpAbout() {
     about->show();
 }
 
+void MixxxMainWindow::slotShowKeywheel(bool toggle) {
+    if (m_pKeywheel == nullptr) {
+        m_pKeywheel = new DlgKeywheel(this, m_pCoreServices->getSettings());
```

Comment:
you can use here parented_ptr to verify that DlgKeywheel is controlled by the Qt object tree.